apps/site/assets/ts/schedule/components/schedule-finder/ScheduleFinderForm.tsx

```diff
@@ -57,73 +58,85 @@ export default ({
+            <span role="alert">Please provide an origin</span>
+          </div>
+        )}
 
-        <label className="schedule-finder__label">
-          Choose an origin stop
-          <SelectContainer error={originError} handleClick={handleOriginClick}>
-            <select
-              className="c-select-custom c-select-custom--noclick"
-              value={selectedOrigin || ""}
-              onChange={e => onOriginChange(e.target.value || null)}
+        <div className="schedule-finder__inputs">
+          <label className="schedule-finder__label">
+            Choose a direction
+            <SelectContainer>
+              <select
+                className="c-select-custom"
+                value={selectedDirection}
+                onChange={e =>
+                  onDirectionChange(parseInt(e.target.value, 10) as DirectionId)
+                }
+              >
+                {validDirections(directionNames).map(direction => (
+                  <option
+                    key={direction}
+                    value={direction}
+                    aria-label={directionNameForId(direction)}
+                  >
+                    {directionNameForId(direction)}
+                  </option>
+                ))}
+              </select>
+            </SelectContainer>
+          </label>
+
+          <label className="schedule-finder__label">
+            Choose an origin stop
+            <SelectContainer
+              error={originError}
+              handleClick={handleOriginClick}
             >
-              <option value="">Select</option>
-              {stopsByDirection[selectedDirection].map(({ id, name }) => (
-                <option key={id} value={id} aria-label={name}>
-                  {name}
-                </option>
-              ))}
-            </select>
-          </SelectContainer>
-        </label>
-      </div>
+              <select
+                className="c-select-custom c-select-custom--noclick"
+                value={selectedOrigin || ""}
+                onChange={e => onOriginChange(e.target.value || null)}
+              >
+                <option value="">Select</option>
+                {stopsByDirection[selectedDirection].map(({ id, name }) => (
+                  <option key={id} value={id} aria-label={name}>
+                    {name}
+                  </option>
+                ))}
+              </select>
+            </SelectContainer>
+          </label>
+        </div>
 
-      <div className="schedule-finder__submit text-right">
-        <input
-          className="btn btn-primary"
-          type="submit"
-          value="Get schedules"
-        />
-      </div>
-    </form>
+        <div className="schedule-finder__submit text-right">
+          <input
+            className="btn btn-primary"
+            type="submit"
+            value="Get schedules"
+          />
+        </div>
+      </form>
+      {isFerryRoute && <hr />}
```

Comment:
Looks like this `<hr />` appears in the schedule finder modal too and it looks super out of place! How about using CSS to draw this line instead when inside a `.schedule-finder-vertical` class?